src/Sql/Sql.Test/ScenarioTests/RefreshSqlInstanceExternalGovernanceCmdletTests.ps1

```diff
@@ -0,0 +1,74 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+<#
+	.SYNOPSIS
+	Tests refresh external governance status
+#>
+
+function Test-RefreshSqlInstanceExternalGovernanceCmdlet ($location = "eastus2euap")
+{
+	#Test scenario
+	try
+	{
+		# ------------------------------ Setup
+		Write-Debug "Creating test MI"
+		$rg = Create-ResourceGroupForTest
+		$rgName = $rg.ResourceGroupName
+		$managedInstance = Create-ManagedInstanceForTest $rg
+		$managedInstanceName = $managedInstance.ManagedInstanceName
+		
+		# ------------------------------ Get the MI
+
+		Write-Debug "Getting the created MI $managedInstanceName"
+		$instance = Get-AzSqlInstance -ResourceGroupName $rgName -Name $managedInstanceName
+		$instanceId = $instance.Id
+		$instanceName = $instance.ManagedInstanceName
+
+		# ------------------------------ Test by passing in the instance object
+
+		Write-Debug "Test by passing in the instance object"
+		$result = Get-AzSqlInstance -ResourceGroupName $rgName -Name $managedInstanceName | Invoke-AzSqlInstanceExternalGovernanceStatusRefresh
+
+		Write-Debug ('$result is ' + (ConvertTo-Json $result))
+		Assert-NotNull $result
+		
+		Assert-AreEqual $result.Status "Succeeded"
+		Assert-AreEqual $result.InstanceName $instanceName
```

Comment:
I would also add 1 get call in these tests. Since we cannot enable external governance using the refresh cmdlet alone (it needs to be enabled on the Purview side, and I think that they don't have PowerShell cmdlets for now, just Portal and raw API requests), I would just get the MI using get cmdlet and check if the ExternalGovernanceStatus is set to False - just as a basic test which check whether the cmdlet actually returns that field